src/sftp/handler_test.go

```diff
@@ -18,6 +17,128 @@ import (
 	"testing"
 )
 
+func Test_NewSFTPHandler_UnableToGetSFTPServerPublicKeyNameSecret_ReturnsError(t *testing.T) {
+    defaultLogger := slog.Default()
```

Comment:
Most of this file looks double tabbed, is that correct?

Reply:
Besides between test functions, yes